phpstan.neon.dist

```diff
@@ -36,7 +36,7 @@ parameters:
 		# This is part of the functionality of this package. Safe to ignore.
 		-
 			message: "`^Call to function method_exists\\(\\) with 'PHPUnit\\W+Framework\\W+(TestCase|Assert)' and '(assert|expect|equalTo)[^']+' will always evaluate to true\\.$`"
-			count: 8
+			count: 7
```

Comment:
https://github.com/Yoast/PHPUnit-Polyfills/pull/239/commits/7fff9a5c3b709f6d633518fd0f9c96cc7852b529 changed it to `7`.

Should this commit's count be changed to `6`?

Reply:
Could you explain why you think that would be needed ? (and in which commit?)

Reply:
🤦‍♀️ oh boo, my mistake. I was looking at the same commit opened in different tabs 🤦‍♀️  

Apologies. This count is correct.